conf/layer.conf

```diff
@@ -29,3 +29,6 @@ LICENSE_PATH += "${LAYERDIR}/licenses"
 
 # Enable early start of Thunder
 DISTRO_FEATURES_append = " compositor thunder"
+
+#Added for testing purpose
+PACKAGECONFIG_pn-wpeframework-buildtype = " debugoptimized"
```

Comment:
We can't do this like this. Could you please add FIXME here?

Reply:
done